templates/page/page.component.inc

```diff
@@ -73,6 +67,24 @@ function europa_preprocess_page(&$variables, $hook) {
   }
   $variables['site_header'] = array('#theme' => array('site_header'));
   $variables['messages'] = $variables['show_messages'] ? array('#theme' => array('status_messages')) : '';
+
+  $variables['page_header'] = array(
+    '#theme' => array('page_header'),
+    '#identity' =>  $variables['site_name'],
+    '#title_prefix' => $variables['title_prefix'],
+    '#title_suffix' => $variables['title_suffix'],
+    '#title' => drupal_get_title(),
+    '#introduction' => NULL,
+    '#meta' => array(),
```

Comment:
I know the component foresees the `meta` item but without mechanism to set it, it is useless.

3 options:

1. Not set it here and let the theme mechanism works naturally; up to the sub-site to set it in a preprocess hook of their theme.
2. Put in place a configuration mechanism like done for the introduction text.
3. Decide that it is only displayed with content (and why not term pages) and select the metadata to display